cape/cape_result.py

```diff
@@ -2194,16 +2197,62 @@ def process_buffers(
     buffer_res.set_column_order(["Process", "Source", "Buffer"])
     buffer_ioc_table = ResultTableSection("Buffer IOCs")
     buffer_body = []
-
+    buffers = []
     for process, process_details in process_map.items():
         count_per_source_per_process = 0
         process_name_to_be_displayed = f"{process_details.get('name', 'None')} ({process})"
         for call in process_details.get("decrypted_buffers", []):
             buffer = ""
-            if call.get("CryptDecrypt"):
+            # Note not all calls have the key name consistent with their capemon api output
+            if call.get("CryptDecrypt"): # Depricated but still used
```

Comment:
this `if` statement feels very repetitive. I think we can optimize this using a `next()` method.